cyctl/internal/get/modules.go

```diff
@@ -47,16 +48,36 @@ func listModules(clientset *client.CyclopsV1Alpha1Client, moduleNames []string)
 		for _, name := range moduleNames {
 			nameSet[name] = struct{}{}
 		}
-		filteredModules = []v1alpha1.Module{}
+		foundModules := make([]v1alpha1.Module, 0)
+		notFoundModules := make([]string, 0)
+
 		for _, module := range modules {
 			if _, found := nameSet[module.Name]; found {
-				filteredModules = append(filteredModules, module)
+				foundModules = append(foundModules, module)
+				delete(nameSet, module.Name)
+			}
+		}
+		for name := range nameSet {
+			notFoundModules = append(notFoundModules, name)
+		}
+		if len(notFoundModules) > 0 {
+			for _, name := range notFoundModules {
+				fmt.Printf("no module found with name: %s\n", name)
 			}
 		}
+		filteredModules = foundModules
+
 	}
 
 	headerSpacing := max(0, longestName-4)
-	fmt.Println("NAME" + strings.Repeat(" ", headerSpacing) + " AGE")
+	output := ""
+	if len(filteredModules) > 0 {
+		output += "NAME" + strings.Repeat(" ", headerSpacing) + " AGE\n"
+	} else {
+		os.Exit(1)
```

Comment:
can you add this check to the end of the function so it prints out the existing modules (if any are found) and then exists with 1?

The output should still look like this:
```
cyctl get modules not-exists exists

no module found with name: not-exists
NAME                 AGE
exists                 28s
```
but should exit with 1